### DISCUSSION TITLE
Silence optimizeUniversalDefaults warning

### DISCUSSION THREAD

**Discussion by mustafa0x** [tailwindlabs/tailwindcss#12210] [Help]
How can I silence this warning? Using vite.

```
warn - You have enabled experimental features: optimizeUniversalDefaults
warn - Experimental features in Tailwind CSS are not covered by semver, may introduce breaking changes, and can change at any time.
```

**Comment by wongjn** [comment]
You could consider disabling `experimental.optimizeUniversalDefaults` by setting it to `false` (or removing it altogether since `false` is the default) in the Tailwind configuration.

Otherwise, if you'd like to keep it enabled, you'd need to see if you can stop `console.warn` calls from outputting to the command line output.

**Reply by mustafa0x** [reply]
> you'd need to see if you can stop console.warn calls from outputting to the command line output.

That's why I'm here :)

**Comment by boycce** [comment]
I'd argue this should be a first-class setting, without this Devtools is a mess.
